todo_app/app.py

```diff
@@ -30,16 +34,7 @@ def hello():
 @app.route('/')
 def index():
     # The index page should go to templates/index.html
-    # unsorted
-    #current_items=get_items()
-
-    # TASK sort by status
-    # sort the list of dictionaries by a dictionay key
-    # Sorting a list of dictionaries raises an error by default if sorted() is used
-    # reference - https://note.nkmk.me/en/python-dict-list-sort/
-    # sorting by status key
-    current_items=sorted(get_items(), key=lambda x: x['status'])  
-    return render_template('index.html', items=current_items)
+    return render_template('index.html', items=get_items(url, trellobid, apikey, apitoken))
```

Comment:
Is there an argument that the Trello environment variables should be captured directly in the `trello_items.py` file, rather than passed through each time?